pkgs/development/tools/vend/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule {
+  pname = "vend";
+  version = "unstable-2020-06-04";
+
+  patches = [./remove_tidy.patch];
+
+  # A permanent fork from master is maintained to avoid non deterministic go tidy
+  src = fetchFromGitHub {
+    owner = "nomad-software";
+    repo = "vend";
+    rev = "dccb39a9f32ea2614b6d5794fa95bec918d90786";
+    sha256 = "1cb1s6ki3pfchqg010n83s6f3wjisjl8i33cgh5sji7glny38pf1";
+  };
+
+  vendorSha256 = null;
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/c00w/vend";
```

Comment:
Nit: out of date link now that you're using upstream plus a patch, rather than a fork, right?

Reply:
Not sure that we want to point to upstream as we're applying a patch that they have refused.

Reply:
Ack, consider this resolved then.